mayday.go

```diff
@@ -62,6 +62,7 @@ func openFile(f File) (*file.MaydayFile, error) {
 
 func main() {
 	pflag.BoolP("danger", "d", false, "collect potentially sensitive information (ex, container logs)")
+	pflag.StringP("output", "o", "", "output file (default: /tmp/mayday{current time}.tar.gz)")
```

Comment:
sounds good!

should we also let people include the date and hostname in their custom names with placeholders like `{hostname}` and `{time}`?